extension/background/languages.js

```diff
@@ -0,0 +1,32 @@
+this.languages = (function() {
+    const exports = {};
+
+    const languageCodes = {
+        Czech: "cs",
+        Danish: "da",
+        Dutch: "nl",
+        English: "en",
+        Finnish: "fi",
+        French: "fr",
+        German: "de",
+        Hungarian: "hu",
+        Italian: "it",
+        Norwegian: "no",
+        Polish: "pl",
+        Portuguese: "pt",
+        Romanian: "ro",
+        Russian: "ru",
+        Slovak: "sk",
+        Slovenian: "sl",
+        Spanish: "es",
+        Swedish: "sv",
+        Turkish: "tr",
+        Ukrainian: "uk",
+    };
+
+    exports.languageCode = function() {
```

Comment:
A bit of naming confusion here: these are the language names (e.g., ,"Italian") not the codes ("it").

Reply:
Fixed.